chasm/archetype.go

```diff
@@ -1,12 +1,13 @@
 package chasm
 
-type Archetype string
+// Archetype is the fully qualified name of the root component of a CHASM execution.
+type Archetype = string
```

Comment:
Should we rename this to `ArchetypeName`? I'm totally okay not doing that FTR.

Reply:
yeah.. I was also debating on this with myself.. On one hand using xxxName and xxxID is similar to what we do for namespace. But on the other hand, we used the word Archetype a lot in our conversation and I want the same name in the code as well. 

hmmm guess I will keep it as is since I also don't see a strong reason to change it.